### DISCUSSION TITLE
Generate an identical image from different angles

### DISCUSSION THREAD

**Discussion by lifeinwork7** [lucidrains/imagen-pytorch#156] [Ideas]
Thank you for the development of technology in machine learning, and especially for your significant contribution to open source projects.
How can we train Imagen or any other neural network, or add some code to generate an identical image from different angles.
For example Imagen can make 1 image from the front side, then use the same coordinates and exactly the same image to generate its additional part from 20 degree left, then 40 degree left and etc., in the end, we can have many pictures of the same object in different angels.
It might help to create 3D models based on photos using other machine learning models like NVIDIA Instant NeRFs
If you have any information or contact for someone to help me it would be appreciated.
Also willing to join development team working on this image generation model from different angles and help with anything from finance to knowledge.
If there is still no such team will try to create myself.
Any information you can give me will be helpful.
Regards

**Comment by maurojosedmjr** [comment]
Hey @lifeinwork7 , how are you?

I'm looking for some guidance on the same problem. I have a frontal image of superhero heads and I need to generate more images from different angles.
I've already tried some algorithms, but without much success.
Did you manage to solve your problem? If so, could you give me some guidance on how to do this better?
Thanks in advance.

**Comment by pinnakkkk** [comment]
Hey @lifeinwork7 & @maurojosedmjr were you able to find a solution? I think something like LOLNeRF: Learn from One Look or Bootstrapped Radiance Field Inversion should work here.
